src/api/app/controllers/webui/groups_controller.rb

```diff
@@ -36,6 +41,18 @@ def create
     redirect_back_or_to root_path, error: "Group can't be saved: #{group.errors.full_messages.to_sentence}"
   end
 
+  def update
+    @group = Group.find_by(title: params[:title])
```

Comment:
What happens if the group title is not found?

Reply:
In other actions we are also just assuming a correct  title comes, but OK, someone can type an inexistent title in the URL, so it doesn't harm to control it.  I'm going to do it in these actions.

Reply:
Please have another look